packages/integration-tests/src/test-project-setup/data_storage_auth_with_triggers.ts

```diff
@@ -228,6 +235,7 @@ class DataStorageAuthWithTriggerTestProject extends TestProjectBase {
     await this.checkLambdaResponse(node16Lambda[0], expectedResponse);
     await this.checkLambdaResponse(funcWithSsm[0], 'It is working');
     await this.checkLambdaResponse(funcWithAwsSdk[0], 'It is working');
+    await this.checkLambdaResponse(funcWithSchedule[0], 'It is working');
```

Comment:
technically this isn't checking that the event trigger is working, just that the lambda can be invoked successfully. I'm not sure how worthwhile it is, but a more comprehensive test might be to create a lambda that sets a value in parameter store (or leaves some trace of execution), set an "every second" schedule for that function, wait a couple seconds after the deployment completes, then check for the execution breadcrumb.

Or maybe a simpler test is to check that the lambda can be invoked (ie what you have here) + check cloudtrail for the invocation event

Reply:
EventBridge has a minimum of 1 minute for rule schedules so we would have to wait for ~1 minute for the event to invoke the lambda.

I was thinking the unit tests checking the event rule is properly added (which I just realized I should also make sure the rule target is properly set) and this E2E test is enough to have confidence this will work. I can explore the 2nd option you suggested.

Reply:
I don't see a way around fully testing this without having the test wait the minimum time it takes for the event to trigger.

Still trying different ways to reduce flakiness with current E2E implementation.

Reply:
I think the 70 second wait have is okay. If it proves to be slow / flaky we can reevaluate the strategy and value of the test. Thanks for investigating this!

Reply:
Thanks! Yeah I was playing with the time considering EventBridge may have a delay of several seconds so 70 seconds seemed like a good number with the minimum time of 1 minute for schedules.